environments/environments.py

```diff
@@ -0,0 +1,94 @@
+from abc import ABC, abstractmethod
+from PIL import Image, ImageDraw
+import numpy as np
+
+
+class Environment(ABC):
+
+    @abstractmethod
+    def set(self, p, q):
+        """Sets initial conditions for physical system
+
+        Args:
+            p (float): generalized momentum
+            q (float): generalized position
+
+        Raises:
+            NotImplementedError: Class instantiation has no implementation
+        """
+        raise NotImplementedError
+
+    def step(self, dt=0.01):
+        """Performs a step in the environment simulator
+
+        Args:
+            dt (float, optional): [description]. Defaults to 0.01.
+        """
+        raise NotImplementedError
+
+    def draw(self):
+        """Returns Caption of the environment state
+        """
+        raise NotImplementedError
+
+    def generateData(self, total_seconds, fps):
```

Comment:
generate_data
Try to follow PEP8 convention of method naming: lowercase with words separated by underscores as necessary to improve readability.